packages/space-opera/src/components/best_practices/constants.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+export const modelViewerTemplate = `
```

Comment:
Any changes you want to make to this template @elalish ?

-- modelviewer and progressbar within the body are replaced with their html elements respectively.

Reply:
Isn't there a script.js file? Can we just `src` that instead of inlining it? The rest looks great; nicely condensed!